analyze_benchmark_data/benchmark_analysis.md

```diff
@@ -0,0 +1,174 @@
+
+``` r
+# 
+raw_quote_file = paste(proj_dir, 
+                    "/benchmark_data/benchmark_quote_table_raw.tsv", 
+                    sep="")
+
+raw_quote_df = read_corenlp_quote_files(raw_quote_file)
+
+head(raw_quote_df)
+```
+
+    ##                                                       file_id     est_speaker
+    ## 1                                          d41586-020-00889-6           Addex
+    ## 2                                          d41586-020-00889-6           Addex
+    ## 3                                          d41586-020-00889-6           Addex
+    ## 4 quantum-technology-probes-ultimate-limits-of-vision-1.17731    Angelo Bassi
+    ## 5 quantum-technology-probes-ultimate-limits-of-vision-1.17731 Anthony Leggett
+    ## 6    ebola-experience-leaves-world-no-less-vulnerable-1.18844      Ashish Jha
+    ##   est_gender canonical_speaker    partial_name
+    ## 1       MALE      spokesperson           Addex
+    ## 2       MALE       Neena Nizar           Addex
+    ## 3       MALE      spokesperson           Addex
+    ## 4       MALE           Unknown    Angelo Bassi
+    ## 5       MALE   Anthony Leggett Anthony Leggett
+    ## 6       MALE        Ashish Jha      Ashish Jha
+    ##                                                                                                          quote
+    ## 1                                                                           The COVID-19 situation is dynamic,
+    ## 2                I feel like we were chugging along on a train and then somebody dropped a huge boulder on it.
+    ## 3 We are now seeing impacts on clinical-trial continuity in all the regions where we conduct clinical studies.
+    ## 4                     something like a superposition of two different perceptions, even if just for an instant
+    ## 5                                               We don't know at what stage it's going to break down — or how.
+    ## 6                                      I honestly doubt that many of these recommendations will be acted upon.
+
+The main columns of interest are `est_gender` and `est_speaker`, which we will compare between lines that have the same `quote` and `file_id`.
+
+### analyzing quote data
+
+First, lets look at the *benchmark* data, to see if there exist any gender disparity evidence. <img src="benchmark_analysis_files/figure-markdown_github/unnamed-chunk-4-1.png" style="display: block; margin: auto;" />
+
+Ok, so we see some signal. Now what does it look like for our estimated gender?
+
+<img src="benchmark_analysis_files/figure-markdown_github/unnamed-chunk-5-1.png" style="display: block; margin: auto;" />
+
+Nice, it looks pretty close. Things called `NO_EST` are when a quote was found, but no gender was able to be estimated. Let's take a closer look at the errors <img src="benchmark_analysis_files/figure-markdown_github/unnamed-chunk-6-1.png" style="display: block; margin: auto;" />
```

Comment:
Interesting that male speakers have higher accuracy than females in your benchmark set. Definitely want to see when you get the full data, but I think there might be a male bias in the types of speakers.

Reply:
There definitely could be a bias -- This may be worth looking at deeper in an expanded benchmark. I could look for another benchmark corpus with gender + quote identified and see if it looks the same.

Reply:
Added this as an issue, so it isn't lost in the future.